docs/projects/ad77681evb/index.rst

```diff
@@ -0,0 +1,259 @@
+.. _ad77681evb:
+
+AD77681EVB HDL project
+=================================================================================
+
+Overview
+---------------------------------------------------------------------------------
+
+The :adi:`AD7768-1`/ :adi:`ADAQ7768-1` is a low power, high performance, Σ-Δ
+analog-to-digital converter (ADC), with a Σ-Δ modulator and digital filter for
+precision conversion of both ac and dc signals. The :adi:`AD7768-1` is a single
+channel version of the AD7768, an 8-channel, simultaneously sampling, Σ-Δ ADC.
+The :adi:`AD7768-1` provides a single configurable and reusable data acquisition
+footprint, which establishes a new industry standard in combined ac and dc
+performance and enables instrumentation and industrial system designers to design
+across multiple measurement variants for both isolated and nonisolated
+applications.
+
+The :adi:`AD7768-1` achieves a 108.5 dB dynamic range when using the low ripple,
+finite impulse response (FIR) digital filter at 256 kSPS, giving 110.8 kHz input
+bandwidth (BW), combined with ±1.1 ppm integral nonlinearity (INL), ±30 µV
+offset error, and ±30 ppm gain error. Wider bandwidth, up to 500 kHz Nyquist,
+204 kHz, −3 dB, is available using the sinc5 filter, enabling a view of signals
+over an extended range. 
+
+A 1.024 MHz sinc5 filter path exists for users seeking an even higher output
+data rate. This path is quantization noise limited; therefore, it is best suited
+for customers requiring minimum latency for control loops or implementing custom
+digital filtering on an external field programmable gate array (FPGA) or digital
+signal processor (DSP).
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD7768-1 <EVAL-AD7768-1>`
+-  :adi:`EVAL-ADAQ7768-1 <EVAL-ADAQ7768-1>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD7768-1`
+-  :adi:`ADAQ7768-1`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+-  :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>` on FMC slot
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The data path and clock domains are depicted in the below diagram:
+
+AD77681EVB
```

Comment:
If there is just one block diagram, there's no need for a subsection

Reply:
Removed.